src/core/File.ts

```diff
@@ -645,9 +645,9 @@ export class File {
 
             if (!this.context.inlineSourceMaps) {
                 delete jsonSourceMaps.sourcesContent;
-            }
-
-            result.outputText = result.outputText.replace("//# sourceMappingURL=module.js.map", "");
```

Comment:
I still think we need to keep the original replacement. As TypeScript adds it to each file. Can we keep it too?